web/src/api/getAppVersion.ts

```diff
@@ -0,0 +1,17 @@
+import { useQuery } from '@tanstack/react-query';
+import { resolvePath } from 'react-router-dom';
+
+async function getVersion(): Promise<{ version: string }> {
+  const response = await fetch(resolvePath('client-version.json').pathname).then(
```

Comment:
Do we need the `resolvePath()` here, or was it just used when extracting from old code?
```suggestion
  const response = await fetch(resolvePath('client-version.json').pathname).then(
```

Reply:
Yeah it does need to so it picks up the root hostname when you're on a zone etc